app/lib/actions/pulp3/orchestration/repository/sync.rb

```diff
@@ -0,0 +1,19 @@
+module Actions
+  module Pulp3
+    module Orchestration
+      module Repository
+        class Sync < Pulp3::Abstract
+          include Actions::Helpers::OutputPropagator
+          def plan(repository, smart_proxy, options)
+            sequence do
+              action_output = plan_action(Actions::Pulp3::Repository::Sync, options).output
+              plan_action(Pulp3::Repository::SaveVersion, repository, action_output[:pulp_tasks])
+              plan_action(Pulp3::Orchestration::Repository::GenerateMetadata, repository, smart_proxy, :force => true)
```

Comment:
do we want to use force => true here?

Reply:
So the CreatePublication action creates a publication if there are no existing publications or if the option is set to true. We do want to create a new publication on sync so force => true should be passed here?

Reply:
ahh, i think the value for 'force' here is if anything changed on the version.  If nothing changed, then no need to generate a new publication.  Maybe we can pass contents_changed here and have GenerateMetadata honor that?

Reply:
Although maybe we should handle that as another issue.